### ISSUE TITLE
Watch later retrieval

### ISSUE THREAD

**Issue by anxdpanic** [jdf76/plugin.video.youtube#495]
- improve instructions/process
- explore page count further
- number of videos required for trending

**Comment by tadly** [comment]
As promised in the other issue I did some more testing in regards to figuring out when the "Watch Later" entry would show up on youtube.com/tv.

Bad news. It did randomly show up all of a sudden. I than started removing one video at a time, hard-reloading the page until I was left with only one video (the one youtube kept suggesting me. I did this on purpose).
Only after I reached 1 video did it **stop** to show up.
Adding a second video caused it to re-appear (suggesting the same video).

The reason I see this as being a issue is because back when it was relevant (me trying to get the id through the addon) I wasn't able to make it work even after adding a bunch of videos.

Seeing as I was the first one to create an issue for this whole ordeal I assume most people haven't had as much _back luck_ as I did.
Mentioning something along the lines of "have more then 10 videos in the list and give it some time (a day or two)" would probably make sens. (I'm pretty sure the time a video has been in WL influences whether or not it shows up)

**TL;DR**
* improve instructions/process
   Mention to have maybe 10+ videos **and** have them in there for a day or two
* explore page count further
   In my case it was pretty high up but increasing it wouldn't hurt
* number of videos required for trending
   Varies (1 was to little, with 2 it came back) and I assume the amount of time a video has been in WL is even more important than the amount

**Comment by anxdpanic** [comment]
Thank you very much for the information and taking the time to gather it. It will definitely help when revising this process.

I was able to find mine with 1 video but that video had been there longer than a month. Which seems to reaffirm times influence.

**Comment by tadly** [comment]
You're very welcome :)

That one video I was referring to must be in my list for about 2-3 days.
Was uploaded on Nov 30th but I'm not sure if I added it at the same day.

**Comment by anxdpanic** [comment]
a321a9f extends the process to 3 steps, with a background progress dialog for user feedback during each step

1. ```Failed to retrieve Watch Later playlist id. Add a few videos to Watch Later via the web/app and retry.```
2. ```Failed to retrieve Watch Later playlist id. To increase the chances of retrieval add 8-10 videos to Watch Later via the web/app and retry.```
3. ```Failed to retrieve Watch Later playlist id. Try again tomorrow.```

Progress dialog: ```Searching for Watch Later... Page %s```

**Comment by tadly** [comment]
The only thing I'd change is to say something along the lines of:
`Failed to retrieve Watch Later playlist id. Try again tomorrow without removing any of the videos.`

**Comment by anxdpanic** [comment]
Good call, and thank you for the feedback. #501 changes the 3rd retry message
